packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -202,6 +202,27 @@ ImageDialog.propTypes = {
   handleClose: PropTypes.func.isRequired,
 };
 
+const isLastBlockImageORCode = (editor) => {
+  const { selection } = editor;
+
+  if (!selection) return false;
+
+  const [currentImageORCodeBlock] = Editor.nodes(editor, {
+    at: selection,
+    match: (n) => n.type === 'image' || n.type === 'code',
+  });
+
+  if (currentImageORCodeBlock) {
+    const [, currentNodePath] = currentImageORCodeBlock;
+
+    const nodeAfter = Editor.after(editor, currentNodePath);
+
+    return !nodeAfter;
```

Comment:
Just for readability could you return `!Boolean(nodeAfter)`? to make it obvious we're returning a boolean